### DISCUSSION TITLE
code/sql/

### DISCUSSION THREAD

**Discussion by giscus** [jeremy-feng/giscus#7] [Announcements]
# code/sql/

https://fengchao.pro/code/sql/

**Comment by hanucodes** [comment]
SQL and TSQL Cheat Sheet
https://helpercodes.com/sql-query-cheatsheet-tutorial/

**Reply by jeremy-feng** [reply]
Thank you sir🥰

**Comment by BILL-SUN521** [comment]
想问一下侧边栏中的icon图标是如何显示的（如pandas左边的图标）？我在mkdocs官方文档中看了一下，在https://squidfunk.github.io/mkdocs-material/insiders/sponsoring-tiers/ 中，Icons, emojis in sidebars说这需要[built-in typeset plugin](https://squidfunk.github.io/mkdocs-material/plugins/typeset/),而这个是sponsor only 的。

**Reply by jeremy-feng** [reply]
对的，我用的是 Insider 版本的。你可以赞助作者获得 Insider 版本的代码，或者等作者的赞助额达到每月 $20,000 后，开源版本应该就支持这个侧边栏图标的功能了。

**Reply by jeremy-feng** [reply]
Material for MkDocs Insider 不需要 Sponsor 即可免费使用了 :smile:

https://squidfunk.github.io/mkdocs-material/blog/2025/11/11/insiders-now-free-for-everyone/

**Comment by BILL-SUN521** [comment]
感谢，回复的有点晚了。您之前在一个大模型微信群里面提到了INTERACTCOMP来评估多轮对话的事情？关于大模型的多轮对话，我今天还刷到一篇ICML 2025 outstanding paper：COLLABLLM，针对用户模糊的输入，来在训练中增强多轮交互的全局效果。不知道会不会对您有所帮助。参考https://zhuanlan.zhihu.com/p/1970154836759643777。


         原始邮件
         
       
发件人：Jeremy Feng ***@***.***&gt;
发件时间：2025年11月12日 17:31
收件人：jeremy-feng/giscus ***@***.***&gt;
抄送：JUNTAO SUN ***@***.***&gt;, Comment ***@***.***&gt;
主题：Re: [jeremy-feng/giscus] code/sql/ (Discussion #7)




Material for MkDocs Insider 不需要 Sponsor 即可免费使用了 😄

https://squidfunk.github.io/mkdocs-material/blog/2025/11/11/insiders-now-free-for-everyone/

**Reply by jeremy-feng** [reply]
多谢分享！还真是有缘分，几个月前在群里讨论的，没想到和你还是群友哈哈！
